apis/beacon/states/validator_balances.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  operationId: "getStateValidatorBalances"
+  summary: "Get validator balances from state"
+  description: "Returns filterable list of validator balances."
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      in: path
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: id
+      description: "Either hex encoded public key (with 0x prefix) or validator index"
+      in: query
+      required: false
+      schema:
+        type: array
+        maxItems: 30
+        uniqueItems: true
+        items:
+          description: "Either hex encoded public key (with 0x prefix) or validator index"
+          type: string
+    - name: status
```

Comment:
do we need this? It requires getting validator and we won't have it in response?

Reply:
I kept the same parameters as for `/validators`.  It's tricky: on the one hand, I can see that it has use (_e.g._ I want the balances of all active validators).  On the other, as you say, it would involve knowing more about the validator and possibly invalidating the whole purpose of the endpoint.

Shall we remove it for now?  We can always add it back at a later date if people need it (and we can find a way to do it sensibly).

Reply:
@mpetrunic I removed the status part of the request.